src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/grammar/java21/ExpectedStringTemplateBasic.txt

```diff
@@ -522,10 +503,8 @@ COMPILATION_UNIT -> COMPILATION_UNIT [2:0]
         |       |       |   |       `--NUM_INT -> 0 [62:17]
         |       |       |   `--RPAREN -> ) [62:18]
         |       |       `--STRING_TEMPLATE_BEGIN -> " [62:20]
-        |       |           |--STRING_TEMPLATE_CONTENT ->  [62:21]
         |       |           |--EMBEDDED_EXPRESSION_BEGIN -> \{ [62:21]
         |       |           |--EMBEDDED_EXPRESSION_END -> } [62:25]
-        |       |           |--STRING_TEMPLATE_CONTENT ->  [62:26]
```

Comment:
These needed to go.

Reply:
Related to https://github.com/checkstyle/checkstyle/pull/14615#discussion_r1518575615